jupyterlab_caip_optimizer/style/index.css

```diff
@@ -5,3 +5,37 @@
   background-image: url('./images/storage_24px.svg');
   background-size: contain;
 }
+
```

Comment:
Should I move these styles out of this file?

Reply:
That SGTM if you have a more specific place to put them